File: Scripts para VScode/Olimpiadas/C/Primo.c
#include<stdio.h>

int main(){

    int numero = 0;
    int eprimo = 1;

    scanf("%d", &numero);

    for(int i = 2; i < numero; i++)
    {

        if(numero % i == 0){

            printf("N");
            eprimo = 0;
            break;

        }

    }
    
    if (eprimo == 1)
    {
        printf("S");
    }
    
}

File: Scripts para VScode/Olimpiadas/Scripts/primo.py
numero = int(input("Escreva um número: "))
eprimo = True
for c in range(2, numero):
    if(numero % c == 0):
        print("Esse número não é primo")
        eprimo = False
        break
if eprimo == True:
    print("Esse número é primo")

